### PR TITLE
Address make vet/fmt complaints

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -60,3 +60,6 @@ fmt:
 
 vet:
 	hack/go-vet.sh ./...
+
+lint:
```
Comment (paulfantom) [inline Makefile]: Add it also to `check` target

**Comment by enxebre** [comment]
overall lgtm once ci/jenkins/e2e is green. Notice that go vet will keep failing on CI until this gets solved https://github.com/openshift/release/pull/1460#issuecomment-420759482

File: Makefile (hunk 1 above)
Comment (ingvagabund) [reply Makefile]: Done

**Pull request merged**